data/AbhayKumarMittal.json

```diff
@@ -0,0 +1,36 @@
+{
+  "name": "Abhay Kumar Mittal",
+  "type": "personal",
+  "displayStatsPublic": true,
+  "bio": "Open Source | Devops | Full Stack Developer,
+  "avatar": "https://avatars.githubusercontent.com/u/76659113?v=4",
```

Comment:
Just a suggestion, you can have your GitHub profile link just like this `https://github.com/MitAbhay.png`

Reply:
> Just a suggestion, you can have your GitHub profile link just like this `https://github.com/MitAbhay.png`

Hey @MitAbhay , you have two files being committed here, either delete the wrong username file or close this pr, you made the required changes in the previous pr too, so that pr works fine
Make sure to make this change in there too